### PR TITLE
Ux 434 admin icons overlap

### PR DESCRIPTION
**Pull request by tangerine007**
434-overlap

No more overlap

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <base href="/"/>
   </head>
   <body>
+  	TEST TEST TEST
```
Comment (chriskacerguis) [inline app/views/welcome/index.html.erb]: ???

**Comment by chriskacerguis** [comment]
Looks like you added a file called

"public/images/Copy of logo.png"

that file is not a good idea, can you please adjust to something more appropriate for web usage?

**Comment by stackus** [comment]
Look like your git client is marking every line as being removed and added. It makes it difficult to tell what lines have actually changed. Can you please correct?

**Comment by tangerine007** [comment]
Oh oops! These changes I made when I was trying to see if my dev enviroment was working, I thought I had removed them. Only sidebar files should have been changed, will fix.

File: public/templates/partials/common/left_sidebar.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (chriskacerguis) [inline public/templates/partials/common/left_sidebar.html]: This looks like an unresolved merge conflict?

**Comment by chriskacerguis** [comment]
Looks to have a lot of issues with this PR.  I'm going to close it, please feel free to address and re-submit.

**Pull request closed without merging**